tests/foreman/cli/test_provisioning.py

```diff
@@ -40,3 +448,51 @@ def test_rhel_pxe_provisioning_on_libvirt():
 
     CaseLevel: Integration
     """
+    LOGGER.info(">>> User/pass: %s/%s" % user_credentials.auth)
+    LOGGER.info(">>> Org: [%s] %s" % (org.id, org.name))
+    LOGGER.info(">>> Loc: [%s] %s" % (loc.id, loc.name))
+    LOGGER.info(">>> Domain: [%s] %s" % (domain.id, domain.name))
+    LOGGER.info(">>> Subnet: [%s] %s" % (subnet.id, subnet.name))
+    LOGGER.info(">>> Lifecycle environment: [%s] %s" % (lce.id, lce.name))
+    LOGGER.info(">>> Base repo: [%s] %s" % (rhel7.id, rhel7.name))
+    LOGGER.info(">>> KS repo: [%s] %s" % (rhel7ks.id, rhel7ks.name))
+    LOGGER.info(">>> Content view: [%s] %s" % (cv.id, cv.name))
+    LOGGER.info(">>> Content view version: [%s]" % (cvv.id))
+    LOGGER.info(">>> Activation key: [%s] %s" % (ak.id, ak.name))
+    LOGGER.info(">>> Compute resource: [%s] %s" % (cr.id, cr.name))
+    LOGGER.info(">>> Architecture: [%s] %s" % (arch.id, arch.name))
+    LOGGER.info(">>> Operating system: [%s] %s %s.%s" % (os.id, os.name, os.major, os.minor))
+    LOGGER.info(">>> Environment: [%s] %s" % (env.id, env.name))
+    LOGGER.info(">>> Host group: [%s] %s" % (hg.id, hg.name))
+    host_ip = '192.168.11.104'   # FIXME Do not hardcode this one
+    host_name = gen_string('alpha').lower()
+    host_pass = gen_string('alpha')
+    LOGGER.info(">>> Host IP: %s" % host_ip)
+    LOGGER.info(">>> Host name: %s" % host_name)
+    LOGGER.info(">>> Host root password: %s" % host_pass)
+    host_parameters = [{'name': 'remote_execution_connect_by_ip', 'value': True}]
+    host_parameters_str = ','.join(["%s=%s" % (i['name'], i['value']) for i in host_parameters])
+    LOGGER.info(">>> Host parameters: %s" % host_parameters_str)
+    parameters = {
+        'build': 'yes',
+        'compute-attributes': 'start=1',
+        'compute-resource-id': cr.id,   # FIXME This should be in host group
+        'hostgroup-id': hg.id,
+        'interface': 'compute_type=bridge, compute_bridge=br0',
```

Comment:
the bridge for automation vlans will be dependent and will come from robottelo.properties. I guess br0 won't work there.
https://github.com/SatelliteQE/robottelo/blob/master/robottelo.properties.sample#L231

Reply:
Thank you - fixed in:
```
[rhel8_provisioning 2205331e] Do not hardcode bridge name
```